## -*- encoding: utf-8 -*-
## This file (moreProblems.sagetex.sage) was *autogenerated* from moreProblems.tex with sagetex.sty version 2020/08/12 v3.5.
import sagetex
_st_ = sagetex.SageTeXProcessor('moreProblems', version='2020/08/12 v3.5', version_check=True)
_st_.endofdoc()
